packages-exp/auth-types-exp/index.d.ts

```diff
@@ -1080,6 +1277,14 @@ export interface User extends UserInfo {
    * @param forceRefresh - Force refresh regardless of token expiration.
    */
   getIdToken(forceRefresh?: boolean): Promise<string>;
+  /**
+   * Returns a de-serialized JSON Web Token (JWT) used to identitfy the user to a Firebase service.
```

Comment:
nit: "deserialized" seems not to be hyphenated. See e.g. https://en.wikipedia.org/wiki/Serialization